### ISSUE TITLE
Docker Hub

### ISSUE THREAD

**Issue by mcnamee** [0xInfection/TIDoS-Framework#115]
Hi,
Thanks for your work on this tool. Would you consider adding to Docker Hub?

It just makes it more accessible (eg. `docker run -it 0xinfection/tidos-framework` to pull and run) and faster to install via Docker (because the image is prebuilt).

Docker Hub makes it pretty simple - you can simply connect your Docker Hub account to the Github repo, and it'll automatically deploy new commits to the Docker image in Docker Hub.

**Comment by VainlyStrain** [comment]
Hey! Sorry for letting you wait this long (I was busy with uni, and forgot the issue due to stress).

I currently am not very familiar with Docker, but I have plans to get into it. Once I have reached a level of confidence, I'll do the linking.

Greetings!

**Comment by VainlyStrain** [comment]
Hey! Unfortunately, I cannot link this repo directly to Docker Hub, since I only see my own userspace. However, I'll try to link a fork of it, which will act as a mirror. Every time the official repo receives a push, I'll update the mirror, and it (should) update the container. There will be no direct modifications to my fork, only the original framework pulls. The additional commits ahead of master you'll see there are the commits I did earlier, that are now merged here in the official repo.

TL:DR, the code will be the same, but for good practice I preferred to let you know.

Greetings!

**Comment by VainlyStrain** [comment]
I think (and hope) it's working now. Could you confirm by `docker pull vainlystrain/tidos-framework:latest`?
